typescript/infra/config/environments/test/interceptor.ts

```diff
@@ -1,21 +1,25 @@
-import {
-  ChainMap,
-  InterceptorConfig,
-  InterceptorType,
-} from '@hyperlane-xyz/sdk';
+import { ChainMap, InterceptorConfig } from '@hyperlane-xyz/sdk';
+import { HookType } from '@hyperlane-xyz/sdk/src/hook/types';
 import { objMap } from '@hyperlane-xyz/utils';
 
-import { chainToValidator, merkleRootMultisig } from './multisigIsm';
+import {
+  chainToValidator,
+  merkleRootMultisig,
+  messageIdMultisig,
+} from './multisigIsm';
 import { owners } from './owners';
 
 export const merkleRoot: ChainMap<InterceptorConfig> = objMap(
   owners,
   (chain, _) => {
     const config: InterceptorConfig = {
       hook: {
-        type: InterceptorType.HOOK,
+        type: HookType.MERKLE_ROOT_HOOK,
       },
-      ism: merkleRootMultisig(chainToValidator[chain]),
+      ism:
+        Math.random() < 0.5
+          ? merkleRootMultisig(chainToValidator[chain])
+          : messageIdMultisig(chainToValidator[chain]),
```

Comment:
imo we shouldn't introduce randomness to the config

Reply:
makes debugging more difficult

Reply:
removed and not needed for test anymore